makelove/web.py

```diff
@@ -0,0 +1,115 @@
+import html
+import json
+import os
+import sys
+import uuid
+from datetime import datetime
+from pathlib import Path
+from zipfile import ZipFile
+from urllib.request import urlretrieve, URLError
+
+from .util import eprint, get_default_love_binary_dir, get_download_url
+
+
+def download_love(version, platform):
+    """
+    Note, mac builds are stored as zip files because extracting them
+    would lose data about symlinks when building on windows
+    """
+    if version != "11.3":
+        eprint("Web builds are only available for Löve 11.3+")
+        sys.exit(1)
+
+    target_path = get_default_love_binary_dir(version, platform)
+    print("Downloading love binaries to: '{}'".format(target_path))
+
+    os.makedirs(target_path, exist_ok=True)
+    try:
+        download_url = "https://github.com/Davidobot/love.js/archive/master.zip"
+        print("Downloading '{}'..".format(download_url))
+        urlretrieve(download_url, os.path.join(target_path, "love.zip"))
+    except URLError as exc:
+        eprint("Could not download löve: {}".format(exc))
+        eprint(
+            "If there is in fact no download on GitHub for this version, specify 'love_binaries' manually."
+        )
+        sys.exit(1)
+    print("Download complete")
+
+
+def render_mustache(tmpl, cx):
```

Comment:
That name confuses me. Is that a thing? Searching in both love.js repos for "mustache" gives me nothing. What is it? Should it maybe have a comment?

Reply:
lovejs uses [mustache templates](https://github.com/janl/mustache.js/) I didn't want to introduce a dependency nor implement the whole template system, so I just implemented the variable interpolation, which is all lovejs uses anyway.

Reply:
Could you add this as a comment? Because I see myself forgetting that and wondering the same thing again in the future.

Reply:
I did add the comment in the patch I attached to my last comment on this PR